### ISSUE TITLE
Swift 3.2 support?

### ISSUE THREAD

**Issue by TheMajorTom** [jonathantribouharet/JTCalendar#360]
I installed JTCalendar on my project using cocoa pods and get several errors over legacy version (like missing JTCalendarDataSource). Is JTCalendar swift 3.2 compatible?

**Comment by jonathantribouharet** [comment]
It works with Swift 4 so I think it works also with Swift 3.2.
You just have to import the module in your controller with:
```
import JTCalendar
```

**Comment by TheMajorTom** [comment]
Hi jonathan i installed by cocoapods, but i havw some issues like jtcalendardatasource, is there a migration wiki to guide us to fix our previous version?  Thank you for your repply

Enviado do meu iPhone

Em 10 de nov de 2017, às 08:58, Jonathan Tribouharet <notifications@github.com<mailto:notifications@github.com>> escreveu:


It works with Swift 4 so I think it works also with Swift 3.2.
You just have to import the module in your controller with:

import JTCalendar

**Comment by jonathantribouharet** [comment]
No, it's a full rewrite, there is no migration guide between the 1.x and the 2.x version.
I think, you have very few change to do to fix the bug you have, maybe something in the bridging header.
Sorry I can't help you more.

**Comment by TheMajorTom** [comment]
ok thank you Jonathan!

________________________________
De: Jonathan Tribouharet <notifications@github.com>
Enviado: sexta-feira, 10 de novembro de 2017 04:34:24
Para: jonathantribouharet/JTCalendar
Cc: TheMajorTom; Author
Assunto: Re: [jonathantribouharet/JTCalendar] Swift 3.2 support? (#360)


No, it's a full rewrite, there is no migration guide between the 1.x and the 2.x version.
I think, you have very few change to do to fix the bug you have, maybe something in the bridging header.
Sorry I can't help you more.
